src/components/LocationTimeZoneSelector.tsx

```diff
@@ -0,0 +1,65 @@
+import { Checkbox } from '@terraware/web-components';
+import { useEffect, useState } from 'react';
+import strings from 'src/strings';
+import { TimeZoneDescription } from 'src/types/TimeZones';
+import { useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
+import TimeZoneSelector from './TimeZoneSelector';
+
+type TimeZoneEntity = {
+  timeZone?: string;
+};
+
+type LocationTimeZoneSelectorProps = {
+  onChangeTimeZone: (tzSelected: TimeZoneDescription | undefined) => void;
+  location: TimeZoneEntity;
+};
+
+export default function LocationTimeZoneSelector(props: LocationTimeZoneSelectorProps): JSX.Element {
+  const [orgTZChecked, setOrgTZChecked] = useState<boolean>(false);
+  const { onChangeTimeZone, location } = props;
+  const defaultTimeZone = useDefaultTimeZone();
+  const [timeZoneEntity, setTimeZoneEntity] = useState<TimeZoneEntity>();
```

Comment:
we could get rid of this and only use location maybe (since onchange will update the location)